torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,673 @@
+        "pears",
+        "miscanthus",
+        "fruits (trees)",
+        "blueberry - high bush",
+        "fababeans",
+        "mixedwood",
+        "hops",
+        "buckwheat",
+        "spring rye",
+        "too wet to seed",
+        "blueberry (undiff)",
+        "triticale (undiff)",
+        "ryegrass",
+        "flaxseed",
+        "birdsfoot trefoil",
+        "ornemental",
+        "plums",
+        "apricots",
+        "spring barley",
+        "sweet clover",
+        "wheat (undiff)",
+        "fescue",
+        "beetroot",
+        "spring triticale",
+        "wheatgrass",
+        "eggplant",
+        "blueberry - low bush",
+        "cranberry",
+        "mustard",
+        "hemp",
+        "blackberry",
+        "dill",
+        "herbs",
+        "cedar",
+        "hazelnut",
+        "raddish",
+        "nuts",
+        "deforested",
+        "winter cereals",
+        "other nursery",
+        "oilseeds (undiff)",
+        "pulses (undiff)",
+        "winter triticale",
+        "poplar",
+        "celery",
+        "hascap",
+        "walnut",
+        "other berries",
+        "peatland - harvested",
+        "other nut",
+        "spelt",
+        "biofuel (undiff)",
+        "artichoke",
+        "other forage",
+        "vetch",
+        "lentils",
+        "quinoa",
+        "other orchards",
+        "forage crops",
+        "meadows",
+        "rye",
+        "oil seeds",
+        "root crops",
+        "groundnuts",
+        "sesame",
+    ]
+
+    features_url = "https://zenodo.org/records/7257688/files/features.tar.gz?download=1"
+    labels_url = "https://zenodo.org/records/7257688/files/labels.geojson?download=1"
+    file_dict = {
+        "features": {
+            "url": features_url,
+            "filename": "features.tar.gz",
+            "extracted_filename": os.path.join("features", "arrays"),
+            "md5": "cad4df655c75caac805a80435e46ee3e",
+        },
+        "labels": {
+            "url": labels_url,
+            "filename": "labels.geojson",
+            "extracted_filename": "labels.geojson",
+            "md5": "bf7bae6812fc7213481aff6a2e34517d",
+        },
+    }
+
+    def __init__(
+        self,
+        root: str = "data",
+        transforms: Optional[Callable[[dict[str, Tensor]], dict[str, Tensor]]] = None,
+        download: bool = False,
+        checksum: bool = False,
+    ) -> None:
+        """Initialize a new CropHarvest dataset instance.
+
+        Args:
+            root: root directory where dataset can be found
+            transforms: a function/transform that takes input sample and its target as
+                entry and returns a transformed version
+            download: if True, download dataset and store it in the root directory
+            checksum: if True, check the MD5 of the downloaded files (may be slow)
+
+        Raises:
+            DatasetNotFoundError: if ``download=False`` but
+                dataset is missing or checksum fails
```

Comment:
```suggestion
            DatasetNotFoundError: If dataset is not found and *download* is False.
```
Just to match the description from other datasets